TricksterBots/Bots/BaseBot.cs

```diff
@@ -208,6 +208,27 @@ protected bool IsCardHigh(Card highestCard, IEnumerable<Card> cardsPlayed)
                    highRank - RankSort(highestCard);
         }
 
+        protected bool IsCardEffectivelyTheSame(Card card, Card target, IEnumerable<Card> knownCards)
+        {
+            if (!IsOfValue(card))
+                return false;
+
+            var suit = EffectiveSuit(target);
+            if (EffectiveSuit(card) != suit)
+                return false;
+
+            var cardRank = RankSort(card);
+            var targetRank = RankSort(target);
+            var highRank = Math.Max(cardRank, targetRank);
+            var lowRank = Math.Min(cardRank, targetRank);
+
+            return lowRank == highRank ||
```

Comment:
Use `highRank - lowRank <= 1` for a quick exit both for being exactly the same or immediately adjacent.